src/src/rx_main.cpp

```diff
@@ -1096,27 +1107,37 @@ void MspReceiveComplete()
 
 static void setupSerial()
 {
+    bool sbusSerialOutput = false;
+
     if (OPT_CRSF_RCVR_NO_SERIAL)
     {
-        // For PWM receivers with no CRSF I/O, only turn on the Serial port if logging is on
+        // For PWM receivers with no serial pins defined, only turn on the Serial port if logging is on
         #if defined(DEBUG_LOG)
-        Serial.begin(firmwareOptions.uart_baud);
+        Serial.begin(serialBaud);
         SerialLogger = &Serial;
         #else
         SerialLogger = new NullStream();
         #endif
+        serialIO = new SerialNOOP();
         return;
     }
 
+    if (config.GetSerialProtocol() == PROTOCOL_SBUS || config.GetSerialProtocol() == PROTOCOL_INVERTED_SBUS)
+    {
+        sbusSerialOutput = true;
+        serialBaud = 100000;
+    }
+    bool invert = config.GetSerialProtocol() == PROTOCOL_SBUS || config.GetSerialProtocol() == PROTOCOL_INVERTED_CRSF;
```

Comment:
Should this be `PROTOCOL_INVERTED_SBUS`? Or is this just super-confusing due to the fact that SBUS is already inverted, so inverted SBUS is actually non-inverted?

Reply:
Super confusing, yes, because SBUS is inverted serial